.github/workflows/preview-release.yml

```
@@ -133,12 +133,14 @@ jobs:
       run: dotnet build "${{ github.workspace }}/${{ env.PROJECT_NAME }}/${{ env.PROJECT_NAME }}.csproj" `
         -c ${{ env.BUILD_CONFIG }} `
         --no-restore
+        -a win-x64
 
     - name: Execute Unit Tests
       run: dotnet test "${{ github.workspace }}/Testing/${{ env.PROJECT_NAME }}Tests/${{ env.PROJECT_NAME }}Tests.csproj" `
         -c ${{ env.BUILD_CONFIG }} `
         --collect:"XPlat Code Coverage" `
         --no-restore
```

Add a back tick to the end of this line for multiline PowerShell

```suggestion
        --no-restore `
```